src/main/java/net/snowflake/ingest/streaming/internal/FileColumnProperties.java

```diff
@@ -84,6 +98,38 @@ class FileColumnProperties {
     this.setDistinctValues(stats.getDistinctValues());
   }
 
+  FileColumnProperties(int fieldId, Statistics<?> statistics) {
+    this.setColumnOrdinal(fieldId);
+    this.setFieldId(fieldId);
```

Comment:
didn't we say fieldId should be 0 for primitive types?

Reply:
The fieldId is set to zero when the `RowbufferStats` is created. Resolve this as this method is no longer needed.